deploy/009-configure-zksync-governance.ts

```diff
@@ -0,0 +1,122 @@
+import { ethers, network } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { ACMCommandsAggregator } from "typechain";
+
+import { SUPPORTED_NETWORKS } from "../helpers/deploy/constants";
+import { guardian } from "../helpers/deploy/deploymentUtils";
+
+const functionSignatures = {
```

Comment:
Would it make sense to extract to a shared file this data (or something else), to be reused in the configurations of multichain governance for other networks (as we are doing at https://github.com/VenusProtocol/governance-contracts/pull/96)?

Reply:
This file is generic, we can use this file for all upcoming remote networks for  the initial configurations of mutichain governance.